distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -222,6 +222,10 @@ void execute(Terminal terminal, String pluginId, boolean isBatch, Environment en
             throw new UserException(ExitCodes.USAGE, "plugin id is required");
         }
 
+        if ("ingest-geoip".equals(pluginId)) {
```

Comment:
This change will be backported to 6.x to avoid breaking users there, and a note will be added to the migration guide. A follow-up will remove this handling in master so that 7.x will indeed report the usual error message.